src/adapter/src/session.rs

```diff
@@ -79,7 +79,10 @@ pub struct Session<T = mz_repr::Timestamp> {
     user: User,
     vars: SessionVars,
     drop_sinks: Vec<ComputeSinkId>,
-    notices: Vec<AdapterNotice>,
+    notices: (
+        mpsc::UnboundedSender<AdapterNotice>,
+        mpsc::UnboundedReceiver<AdapterNotice>,
+    ),
```

Comment:
Seems marginally clearer to give both of these names?

```suggestion
    notices_tx: mpsc::UnboundedSender<AdapterNotice>,
    notices_rx: mpsc::UnboundedReceiver<AdapterNotice>,
```